packages/tools/babylonServer/public/index.html

```diff
@@ -1,43 +1,133 @@
 <!doctype html>

```

Comment:
Please do not push that file

Everything else is fine

Reply:
Sure, it's actually the first sentence in the PR comment. 😉

I just included it here so a reviewer could easily test the functionality.

Reply:
In fact, you can create a regular PG to test the new feature, as we can test it with the PR merged thanks to a temporary link generated by the CI.